app/controllers/admin_scope/articles_controller.rb

```diff
@@ -3,7 +3,6 @@ class ArticlesController < BaseController
     before_action -> { authorize article }, except: :index
```

Comment:
please use whitelist instead of exception
```suggestion
    before_action -> { authorize article }, only: ["some_actions"]
```